docs/articles/infrastructure/ci-cd/build-slaves.rst

```diff
@@ -6,7 +6,11 @@ Build Slaves
 Running a public slave
 ----------------------
 
-In order to run a slave which the master can reach via SSH you still need to prepare some things. First install docker from `dockers own repositories`_. (The package in Ubuntus own repository is old and outdated):
+It is recommended to use Ubuntu LTS (which as of writing is 16.04) on public slaves.
```

Comment:
I disagree with this. I would rather recommend to run the latest release while developing.

Reply:
But even so, we can describe the issue with 16.04 anyway, since I suppose it will be a common host system for users.

Reply:
Changed.